samples/excel/30-events/events-comment-event-handler.yaml

```diff
@@ -0,0 +1,195 @@
+
+                // Note that an InvalidArgument error will be thrown if multiple cells are passed to `comment.add`.
+                sheet.comments.add("A2", "TODO: add data.");
+                await context.sync();
+            });
+        }
+
+        async function editComment() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Comments");
+                const comment = sheet.comments.getItemAt(0);
+                comment.content = "PLEASE add headers here.";
+                await context.sync();
+            });
+        }
+
+        async function deleteComment() {
+            await Excel.run(async (context) => {
+                context.workbook.comments.getItemByCell("Comments!A2").delete();
+                await context.sync();
+            });
+        }
+
+        async function setup() {
+            await Excel.run(async (context) => {
+                context.workbook.worksheets.getItemOrNullObject("Comments").delete();
+                const sheet = context.workbook.worksheets.add("Comments");
+
+                sheet.activate();
+                await context.sync();
+            });
+        }
+
+        /** Default helper for invoking an action and handling errors. */ 
+        async function tryCatch(callback) {
+            try {
+                await callback();
+            } catch (error) {
+                // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+                console.error(error);
+            }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample shows how to add, edit, and remove comments.</p>
+        </section>
+        <section class="setup ms-font-m">
+            <h3>Setup</h3>
+            <button id="setup" class="ms-Button">
+                <span class="ms-Button-label">Create a worksheet</span>
+            </button>
+            <p></p>
+            <button id="registerEventHandlers" class="ms-Button">
+                <span class="ms-Button-label">Register event handlers</span>
+            </button>
+        </section>
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <p>To better view the comment changes, open the Comments pane by selecting <b>Show Comments</b> from the
+                <b>Review</b> tab.</p>
+            <button id="add-comment-to-selected-cell" class="ms-Button">
+                <span class="ms-Button-label">Add comment to selected cell</span>
+            </button>
+            <p></p>
+            <button id="add-comment-to-cell" class="ms-Button">
+                <span class="ms-Button-label">Add comment to A2</span>
+            </button>
+            <p></p>
+            <button id="edit-comment" class="ms-Button">
+                <span class="ms-Button-label">Edit first comment</span>
```

Comment:
If you take the earlier note of dialing down to one "add" action, we can simplify these buttons to just be "Add comment", "Edit comment", and "Remove comment", instead of giving more specifics.